packages/graphql-playground-react/tsconfig.build.json

```diff
@@ -1,20 +1,7 @@
 {
+  "extends": "./tsconfig.json",
```

Comment:
In order not to repeat ourselves, I made this file extend the main tsconfig and only override the things we want changed.

Reply:
good i was gonna do that